user-guide/Advanced_Functionality/Security/About_DMS_Security/Encryption_in_DataMiner.md

```diff
@@ -0,0 +1,89 @@
+
+DataMiner encrypts the following secrets using this technique:
+
+- Password parameters
+- [Credential libraries](xref:Managing_predefined_sets_of_credentials_for_SNMP_authentication)
+- Database passwords
+- Element passwords
+- Replication passwords
+
+## In-Transit Encryption
+
+Here we can distinguish several communication flows:
+
+- DataMiner Cube
+- DataMiner Web Apps & API
+- Inter-DataMiner
+- Data acquisition
+- Cassandra
+- Elasticsearch
+- NATS
+
+### DataMiner Cube
+
+DataMiner Cube will communicate with the DataMiner backend over .NET Remoting. By default, this is encrypted using the Rijndael (256-bit key, CBC mode) algorithm. The encryption key is negotiated over a 2048-bit RSA-secured communication channel.
+
+### DataMiner Web Apps & API
+
+By default, the DataMiner Web Apps (Reports, Dashboards, Monitoring,...) are served over HTTP, which is unencrypted. We recommend [enabling HTTPS and disabling HTTP](xref:Setting_up_HTTPS_on_a_DMA) to ensure all traffic is encrypted.
+
+### Inter-DataMiner
+
+When a DataMiner System consists of multiple DataMiner nodes, inter-node communication flows through a .NET Remoting channel. By default, this is encrypted using the Rijndael algorithm (256-bit key, CBC mode). The encryption key is negotiated over a 2048-bit RSA-secured communication channel.
+
+### Data acquisition
+
+Depending on the type of device, DataMiner supports different communication protocols:
+
+| Protocol | Description |
+| -------- | ----------- |
+| SSH | DataMiner supports a broad list of SSH cipher, host key algorithms and key exchange algorithms. For more information, see [SSH support in DataMiner](xref:ConnectionsSerialSecureShell#ssh-support-in-dataminer). |
+| SNMP | We recommend interfacing with SNMP devices by using SNMPv3, which provides authentication and privacy. |
+| HTTP(S) | We recommend interfacing with devices or APIs over [HTTPS](xref:ConnectionsHttpHttps), which uses TLS encryption. |
+| TCP/IP | DataMiner can poll devices over a TLS-encrypted TCP/IP channel. See [Enabling TLS encryption](xref:Enabling_TLS_encryption). |
+
+### Cassandra
+
+For DataMiner Systems configured to use a Cassandra database, it is possible to [enable TLS encryption between DataMiner and the Cassandra cluster](xref:DB_xml#enabling-tls-on-the-cassandra-database-connection). Please refer to the official Cassandra documentation on [enabling TLS encryption](https://docs.datastax.com/en/cassandra-oss/3.x/cassandra/configuration/secureSSLClientToNode.html).
+
+We recommended changing the default Cassandra credentials.
+
+### Elasticsearch
+
+For DataMiner Systems configured to use an Elasticsearch database, it is recommended to enable HTTPS. Please refer to the official Elasticsearch documentation on [enabling TLS encryption](https://www.elastic.co/blog/configuring-ssl-tls-and-https-to-secure-elasticsearch-kibana-beats-and-logstash).
```

Comment:
Avoid using an infinitive after "recommend". Possibly rephrase as "we recommend enabling HTTPS"